src/common/Utilities.py

```diff
@@ -463,3 +463,29 @@ def str_to_int_list(str_list):
             int_item = list(range(strt, stop + 1))
         int_list = int_list + int_item
     return int_list
+
+def dot_product_test(operator, num_tests = 20, max_err = 10e-5):
+    '''
+    Test if a given operator is adjoint.
+    The operator needs to have been already set_up() with valid AcquisitionModel and ImageData objects.
```

Comment:
```suggestion
    The operator needs to have been already set_up() with valid objects.
```
I generalised this a bit, as the text wouldn't have applied to registration etc